psi_cardinality/cpp/BUILD

```diff
@@ -40,7 +41,11 @@ cc_library(
 cc_test(
     name = "bloom_filter_test",
     srcs = ["bloom_filter_test.cpp"],
-    linkopts = PSI_CARDINALITY_DEFAULT_LINKOPTS,
+    linkopts = 
+        select({
```

Comment:
I think you can move the select into the list definition above. Also, the current version will only select x86_64 Linux. There doesn't seem to be a config_setting to select for Linux independently of the architecture. We can either add our own using the platform constraint (as suggested [here](https://github.com/bazelbuild/bazel/issues/11107)), or make Linux the default case. Something like that maybe?
```
PSI_LINKOPTS = select({
     "@bazel_tools//src/conditions:darwin": []
     "//conditions:default": ["-lrt"]
})
```

Reply:
I like this simple example you just wrote. Will update.